benchmark_runner/common/clouds/IBM/ibm_operations.py

```diff
@@ -0,0 +1,157 @@
+
+import datetime
+from typeguard import typechecked
+from tenacity import retry, stop_after_attempt
+
+from benchmark_runner.common.logger.logger_time_stamp import logger_time_stamp, logger, datetime_format
+from benchmark_runner.main.environment_variables import environment_variables
+from benchmark_runner.common.remote_ssh.remote_ssh import ConnectionData, RemoteSsh
+from benchmark_runner.common.ocp.create_ocp_resource import CreateOcpResource
+from benchmark_runner.common.oc.oc import OC
+from benchmark_runner.common.github.github_operations import GitHubOperations
+
+
+class IBMOperations:
+    """
+    This class is responsible for all IBM cloud operations, all commands run on remote provision IBM host
+    """
+
+    @typechecked
+    def __init__(self, user: str):
+        self.__environment_variables_dict = environment_variables.environment_variables_dict
+        self.__ibm_api_key = self.__environment_variables_dict.get('ibm_api_key', '')
+        self.__ibm_oc_user = self.__environment_variables_dict.get('provision_oc_user', '')
+        # FUNC or PERF
+        self.__ocp_env_flavor = self.__environment_variables_dict.get('ocp_env_flavor', '')
+        self.__install_ocp_version = self.__environment_variables_dict.get('install_ocp_version', '')
+        self.__num_ocs_disks = int(self.__environment_variables_dict.get('num_ocs_disk', ''))
+        self.__connection_data = ConnectionData(host_name=self.__environment_variables_dict.get('provision_ip', ''),
+                                                user_name=user,
+                                                port=int(self.__environment_variables_dict.get('provision_port', '')),
+                                                timeout=int(
+                                                    self.__environment_variables_dict.get('provision_timeout', '')),
+                                                ssh_key=self.__environment_variables_dict.get('provision_ssh_key', ''))
+        self.__remote_ssh = RemoteSsh(self.__connection_data)
+        self.__github_operations = GitHubOperations()
+
+    def __get_kubeadmin_password(self):
+        """
+        This method return kubeadmin password
+        :return:
+        """
+        return self.__remote_ssh.run_command(f'cat /home/{self.__ibm_oc_user}/clusterconfigs/auth/kubeadmin-password')
```

Comment:
That's making some very specific assumptions about the location of the kubeadmin-password file.  Can't you do it by taking the KUBECONFIG, stripping off the file component, and adding `kubeadmin-password`?

Reply:
I can add it as environment variable and put it on secret, is it ok ?